src/modules/account/components/notifications/notifications-templates.tsx

```diff
@@ -6,10 +6,14 @@ import {
   MarketProgress,
   formatTime
 } from "modules/common-elements/progress";
-import { MARKET_STATUS_MESSAGES } from "modules/common-elements/constants";
+import { SubmitTextButton } from "modules/common-elements/buttons";
```

Comment:
this needs to be roboto mono

Reply:
<img width="94" alt="Screen Shot 2019-04-01 at 3 16 46 PM" src="https://user-images.githubusercontent.com/6775839/55363215-30f71100-5491-11e9-9c7b-29f13d57ed15.png">

Reply:
good catch!